internal/assets/manifests/istio-cni/templates/daemonset.yaml

```diff
@@ -71,6 +71,8 @@ spec:
                   key: cni_network_config
```

Comment:
Will try to followup more closely on this but I was assuming the filename would need to be `name-with-revision` as well.  In the chat test info you showed I saw the filename had rev appended so probably this works with this changeset so I'll try to grok whether there's any missing conditions.

Reply:
The CNI_CONF_NAME value is not generated, but it specified by the user in the icp spec, so it is up to the user the set it properly.

Reply:
Also, it is the actual filename of the config on the disk, it is not the `name` property inside the config.